src/map/latent_effect_container.cpp

```diff
@@ -712,6 +722,18 @@ bool CLatentEffectContainer::ProcessLatentEffect(CLatentEffect& latentEffect)
     case LATENT_WEAPON_SHEATHED:
         expression = m_POwner->animation != ANIMATION_ATTACK;
         break;
+    case LATENT_TARGET_EVEN_MATCH_LOWER:
+    {
+        if (m_POwner->GetBattleTargetID()) {
```

Comment:
{ on newline

Reply:
Not sure why there's a check on GetBattleTargetID when we check the pointer later. Could this be simplified to just checking the pointer? Unless I'm missing something.